### DISCUSSION TITLE
Get all Cookies

### DISCUSSION THREAD

**Discussion by futureweb** [spatie/browsershot#626] [Ideas]
Function to get all Cookies from requested Page using 'Network.getAllCookies'
Use Case: Cookie Consent Manager with Cookie Crawler for GDPR / EU Cookie Law ... so possibility to get all Cookies back is needed (optional - with Info who (html / js) set the Cookie, if possible)

**Comment by futureweb** [comment]
Any News on this? Unfortunately my Pull Request never made it into Browsershot ... and I guess with current Version it's not working anymore ... ?! Haven't had the time to check.

https://github.com/spatie/browsershot/pull/350

**Comment by carestad** [comment]
Would really love to have an option to get the cookies somehow. Is it doable in the later versions of puppeteer and/or Browsershot somehow?
